working_with_fhir/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+org = "healthcare_samples"
+name = "working_with_fhir"
+version = "0.1.0"
+distribution = "2201.4.0"
```

Comment:
why 2201.4.0? Currently we are using 2201.4.1

Reply:
Ack.